.github/workflows/generate-manifest.yml

```diff
@@ -36,7 +36,11 @@ jobs:
           [ "$VERSION" == "main" ] && VERSION=$(echo ${{ github.sha }} | cut -c1-8)
           echo IMAGE_ID=$IMAGE_ID
           echo VERSION=$VERSION
-          make docker-build docker-push split-installer IMG=$IMAGE_ID:$VERSION
+          if [[ "${{ github.repository_owner }}" == "${{ github.actor }}" && "${{ github.ref }}" == "refs/heads/main" ]]; then
+            make docker-build docker-push split-installer IMG=$IMAGE_ID:$VERSION
+          else
+            make docker-build split-installer IMG=$IMAGE_ID:$VERSION
+          fi
```

Comment:
Use `if: github.event_name != 'pull_request'` on the step instead. If we want to try to build the docker image on pull requests then that should be a separate step.